doc/apidoc/smac.epm.random_epm.rst

```diff
@@ -0,0 +1,7 @@
+smac\.epm\.random\_epm module
+=============================
+
+.. automodule:: smac.epm.random_epm
```

Comment:
@mlindauer @KEggensperger as a general question, do we want the documentation to show all methods which are inherited from the base class? I think this would be a great addition to the docs.